core/src/test/java/org/htsjdk/core/utils/PathSpecifierUnitTest.java

```diff
@@ -0,0 +1,254 @@
+package org.htsjdk.core.utils;
+
+//TODO: NAMING: is there any useful distinction between Unit/Integration tests in this repo ?
```

Comment:
I think that we should set this naming conventions, as we did not though about that yet.